config.toml.in

```diff
@@ -37,10 +37,22 @@ type = "file"
   # Specifies the key length (in bytes) to use for AES.
   # Currently supported values: 32, 24, 16 for 256bit, 192bit, 128bit respectively.
   #keylen = 32
-  # The hash function to use for PBKDF2.
-  # Currently supported values: "sha512", "sha256".
-  #pbkdf2hash = "sha512"
-  # The number of iterations used to derive the key from your password.
-  #pbkdf2iterations = 100000
-  # Specifies the salt length (in bytes) for the PBKDF2.
-  #pbkdf2saltlen = 16
+  # Specifies the key derivation function to use.
+  # Currently supported values: "pbkdf2", "scrypt".
+  #keyderivation = "pbkdf2"
+    # Settings for the "pbkdf2" key-derivation type
+    #[encryption.aes_gcm.pbkdf2]
```

Comment:
Is this really the best location for the `pbkdf2` settings? Maybe we should move them to `[encryption.pbkf2]` so other encryption types can share the same key derivation config.
IMO using `[encryption.pbkf2]` would be confusing, as we also have `[encryption.openpgp]` and `[encryption.aes_gcm]`, but `pbkdf2` is no "encryption type".